### PR TITLE
add logical_xor Tensorflow frontend

### PR DESCRIPTION
**Pull request by ntg2208**

### CODE REVIEW

**Comment by ntg2208** [comment]
PR for #2603

File: ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py [hunk 1]
```diff
@@ -207,6 +207,8 @@ def _permute_dims_helper(draw):
 
 
 # permute_dims
+
```
Comment (Unevilicorn) [inline ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py:210]: There shouldn't be spaces between the comment and the function, same with the other functions in this file

**Review by Unevilicorn — CHANGES_REQUESTED** [review]
Looks good! Just a small change then you're good to go.

**Comment by Unevilicorn** [comment]
All merged :+1:

**Pull request merged**